common/reviews/api/ts-command-line.api.md

```diff
@@ -155,19 +156,23 @@ export abstract class CommandLineParameterProvider {
     defineStringParameter(definition: ICommandLineStringDefinition): CommandLineStringParameter;
     // @internal
     protected abstract _getArgumentParser(): argparse.ArgumentParser;
-    getChoiceListParameter(parameterLongName: string): CommandLineChoiceListParameter;
-    getChoiceParameter(parameterLongName: string): CommandLineChoiceParameter;
-    getFlagParameter(parameterLongName: string): CommandLineFlagParameter;
-    getIntegerListParameter(parameterLongName: string): CommandLineIntegerListParameter;
-    getIntegerParameter(parameterLongName: string): CommandLineIntegerParameter;
+    getChoiceListParameter(parameterLongName: string, parameterScope?: string): CommandLineChoiceListParameter;
```

Comment:
The actual long name of the parameter would be `--scope:thing`, not `--thing`.  Can I look it up with `parameterLongName`=`--scope:thing`?  I understand the scenario of searching for `--thing` within a particular scope, but for a caller who starts with the `--scope:thing` string input, it would be weird if they had to parse it into its parts and pass them as separate arguments.